ts/screens/profile/SecurityScreen.tsx

```diff
@@ -37,24 +31,39 @@ import {
   trackBiometricActivationAccepted,
   trackBiometricActivationDeclined
 } from "../onboarding/biometric&securityChecks/analytics";
+import { IOScrollViewWithLargeHeader } from "../../components/ui/IOScrollViewWithLargeHeader";
+import { FAQsCategoriesType } from "../../utils/faq";
 
 const contextualHelpMarkdown: ContextualHelpPropsMarkdown = {
   title: "profile.preferences.contextualHelpTitle",
   body: "profile.preferences.contextualHelpContent"
 };
 
+const FAQ_CATEGORIES: Array<FAQsCategoriesType> = [
+  "profile",
+  "privacy",
+  "authentication_SPID"
+];
+
 const SecurityScreen = (): React.ReactElement => {
   const dispatch = useIODispatch();
   const isFingerprintEnabled = useIOSelector(isFingerprintEnabledSelector);
-  const navigation =
-    useNavigation<IOStackNavigationProp<IdPayCodeParamsList>>();
-  const commonNavigation = useIONavigation();
-  const isScreenReaderEnabled = useScreenReaderEnabled();
-  const [isFingerprintAvailable, setIsFingerprintAvailable] = useState(false);
-  const isIdPayEnabled = useIOSelector(isIdPayEnabledSelector);
   const isIdPayCodeOnboarded = useIOSelector(isIdPayCodeOnboardedSelector);
+  const isIdPayEnabled = useIOSelector(isIdPayEnabledSelector);
+  const navigation = useIONavigation();
+  const [isFingerprintAvailable, setIsFingerprintAvailable] = useState(false);
```

Comment:
```suggestion
  const [isBiometricDataAvailable, setIsBiometricDataAvailable] = useState(false);
```
Just a suggestion, but the current variable name is not semantically correct because it doesn't include other biometric data (e.g. FaceID data).

Reply:
@dmnplb That's true, I'll apply this change!